test/e2e/storage/persistent_volumes.go

```diff
@@ -41,15 +45,15 @@ import (
 
 // Validate PV/PVC, create and verify writer pod, delete the PVC, and validate the PV's
 // phase. Note: the PV is deleted in the AfterEach, not here.
-func completeTest(f *framework.Framework, c clientset.Interface, ns string, pv *v1.PersistentVolume, pvc *v1.PersistentVolumeClaim) {
+func completeTest(f *framework.Framework, c clientset.Interface, ns string, pv *v1.PersistentVolume, pvc *v1.PersistentVolumeClaim, act func(c clientset.Interface, t *framework.TimeoutContext, ns string, pvc *v1.PersistentVolumeClaim, command string) error) {
 	// 1. verify that the PV and PVC have bound correctly
```

Comment:
Does this test case reproduces the issue reliably?

Reply:
hard to say.  some operation orders required to reproduce the problem, this test case can't guarantee it.

Reply:
Agree, not reliable, only add a e2e test coverage to prevent regression